FreeRTOS/FreeRTOS-Kernel
# Check: improve verbosity of url verifier

Improve verbosity of url verifier

Description
-----------
Improve verbosity of url verifier to display warning for redirects an error for 400 results

Test Steps
-----------
Checks in github should display the extra information



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
